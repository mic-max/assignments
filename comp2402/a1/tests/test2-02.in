Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar
Monkey
Cantalope
Jaguar