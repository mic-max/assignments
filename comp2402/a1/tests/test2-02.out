Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Monkey
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Cantalope
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
Jaguar
